plugins/woocommerce/client/admin/client/homescreen/activity-panel/orders/index.js

```diff
@@ -275,7 +275,7 @@ function OrdersPanel( { unreadOrdersCount, orderStatuses } ) {
 		const { getOrders, hasFinishedResolution, getOrdersError } =
 			select( ordersStore );
 		// eslint-disable-next-line @wordpress/no-unused-vars-before-return
-		const { getItems } = select( ITEMS_STORE_NAME );
+		const { getItems } = select( itemsStore );
```

Comment:
Maybe moving this line closer to where the variable will be used to eliminate the `eslint-disable-next-line` annotation.